src/containers/Application/ApplicationList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Container, Table, List, Label } from 'semantic-ui-react'
+import { Loading, FilterList } from '../../components'
+import { useRouter } from '../../utils/hooks/useRouter'
+import useListApplication from '../../utils/hooks/useListApplications'
+
+const ApplicationList: React.FC = () => {
+  const { query } = useRouter()
+  const { error, loading, applications } = useListApplication()
+
+  return error ? (
+    <Label content="Problem to load applications list" error={error} />
+  ) : loading ? (
+    <Loading />
+  ) : (
+    <Container>
+      <FilterList />
+      {Object.keys(query).length > 0 && <h3>Query parameters:</h3>}
+      <List>
+        {Object.entries(query).map(([key, value]) => (
+          <List.Item key={`app_selected_parameter_${value}`} content={key + ' : ' + value} />
+        ))}
+      </List>
+      {applications && applications.length > 0 && (
+        // TODO: Implement sortable columns -> Requires reducer
+        <Table sortable stackable selectable>
+          <Table.Header>
+            <Table.Row>
+              {Object.entries(applications[0]).map(([key]) => (
+                <Table.HeaderCell key={`app_header_${key}`}>{key}</Table.HeaderCell>
+              ))}
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            {applications.map((application, index) => (
+              <Table.Row key={application.serial}>
+                {Object.values(application).map((value) => (
+                  <Table.Cell key={`app_${index}_${value}`}>{value}</Table.Cell>
+                ))}
+              </Table.Row>
```

Comment:
```suggestion
              <Table.Row key={application.serial}>
                {Object.entries(application).map(([key, value]) => (
                  <Table.Cell key={`app_${index}_${key}`}>{value}</Table.Cell>
                ))}
              </Table.Row>
```
I think this would be better, as we need to use the `key` for #67 anyway.